.github/workflows/inspektor-gadget.yml

```diff
@@ -1531,6 +1536,84 @@ jobs:
       run: |
         perl tools/check-readme.pl ./kubectl-gadget README.md
 
+
+  publish-test-reports:
+    name: Publish test reports
+    # level: 5
+    needs:
+      - test-integration-minikube
+      - test-integration-aks
+      - test-integration-aro
+      - test-integration-k8s-ig
+      - test-integration-non-k8s-ig
+    runs-on: ubuntu-latest
+    # Skip this job when running on a fork or a PR from a fork.
+    if: always() && (
+      (github.event_name == 'push' && github.repository == 'inspektor-gadget/inspektor-gadget') ||
+      (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'inspektor-gadget/inspektor-gadget'))
+    steps:
+    - uses: actions/checkout@v4
+    - name: Get all reports
+      uses: actions/download-artifact@v3
+      with:
+        name: "test-reports"
+    - name: Store test reports
+      shell: bash {0}
+      run: |
```

Comment:
Perhaps move this to something under https://github.com/inspektor-gadget/inspektor-gadget/tree/main/tools to avoid adding more lines to this file?

Reply:
Good idea! I will do it!

Reply:
Done.